src/nyc_trees/apps/home/templates/flatpages/default.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}
+
+{% block content %}
```

Comment:
I put some pretty basic text and it was smushed up against the navbar. It might be useful to add some mandatory padding so that not every flatpage has to do their own padding. Probably makes sense to play with it more when we have some data though.